### PR TITLE
Fix the Bay County scraper for multiple associated cases 

### PR DESCRIPTION
**Pull request by lzoz**
To fix issue #94 where the scraper fails when the Case Numbers in Bay County represent multiple separate cases.

### CODE REVIEW

File: Counties/Florida/Bay County/Scraper/Scraper.py [hunk 1]
```diff
@@ -301,7 +310,10 @@ def search_portal(case_number):
     Performs a search of the portal from its home page, including selecting the case number input, solving the captcha
     and pressing Search. Also handles the captcha being solved incorrectly
     :param case_number: Case to search
-    :return: True if a valid case was found, False if not.
+    :return: A dictionary with 3 keys: 'found', 'count', 'cases'.
```
Comment (mcsaucy) [inline Counties/Florida/Bay County/Scraper/Scraper.py:313]: Would a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) be a better fit for this?

Comment (mcsaucy) [reply Counties/Florida/Bay County/Scraper/Scraper.py:313]: Actually, couldn't you just return `cases`? You could then derive `found` by checking the truthiness of `cases` with `if cases` or whatever, and you could get `count` via `len(cases)`.

File: Counties/Florida/Bay County/Scraper/Scraper.py [hunk 2]
```diff
@@ -101,7 +101,7 @@ def begin_scrape():
         last_case_number = ScraperUtils.get_last_csv_row(output_file).split(',')[3]
         print("Continuing from last scrape (Case number: {})".format(last_case_number))
         last_year = 2000 + int(str(last_case_number)[:2])  # I know there's faster ways of doing this. It only runs once ;)
-        last_case = int(str(last_case_number)[-6:])
+        last_case = int(str(last_case_number if last_case_number.isnumeric() else last_case_number[:-4])[-6:])
```
Comment (mcsaucy) [inline Counties/Florida/Bay County/Scraper/Scraper.py:104]: This is pretty hard to grok. Could you break this into a couple lines?

File: Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py [hunk 3]
```diff
@@ -350,3 +350,15 @@ def parse_out_path(directory, filename, extension):
...
+
+def get_associated_cases(driver):
+    """
+    When a  case number is associated with multiple cases, the search portal returns all those cases.
+    This function returns all the associated case numbers.
+    :param driver: Selenium driver
+    :return: A set of case numbers
+    """
+    elems = driver.find_elements_by_class_name('sorting_1')
+    case_nums = set(set([e.text for e in elems]))
```
Comment (mcsaucy) [inline Counties/Florida/Bay County/Scraper/utils/ScraperUtils.py:363]: I think you're accidentally double-`set`-ing here.

**Comment by OscarVanL** [comment]
Great job, I haven't tested it yet but looks like a good solution to the problem. Thanks for the contribution :)

**Comment by mcsaucy** [comment]
Can you squash the commits into 1 as requested in CONTRIBUTING.md? I don't have access to merge changes, but I imagine it may come up. Thanks again for taking the initiative!

**Comment by OscarVanL** [comment]
Just tested this PR out and it works very well. Nice job :)

**Pull request merged**